Allure.NUnit/Core/AllureExtensions.cs

```diff
@@ -184,16 +204,5 @@ public static T WrapInStep<T>(this AllureLifecycle lifecycle, Func<T> func, stri
                 throw;
             }
         }
-
-        /// <summary>
-        /// AllureNUnit AddScreenDiff wrapper method.
-        /// </summary>
-        public static void AddScreenDiff(this AllureLifecycle lifecycle, string expected, string actual, string diff)
```

Comment:
It matches the signature of a new AllureLifecycle's instance method: `AllureLifecycle AllureLifecycle.AddScreenDiff(string, string, string)`, hence will never be called via the member access syntax as the method resolution prefers instance methods to extension methods.
It also does the same thing: tries to access the current test result and add a diff to it.

It can still be called via the static method call syntax though, so your point is valid. I've added it back as obsoleted.